functional_tests/scylla_operator/test_functional.py

```diff
@@ -775,6 +775,7 @@ def test_operator_managed_tls(db_cluster: ScyllaPodCluster, tmp_path: path.Path)
         log.debug(file)
         log.debug(file.read_text())
 
+    db_cluster.nodes[0].refresh_ip_address()
```

Comment:
Why it was added?

Reply:
see this failure:

https://jenkins.scylladb.com/job/sct-github-PRs-scan/job/scylla-cluster-tests/job/PR-5494/9/execution/node/117/log/

could be that now it's not needed, and that run was before merging the ip tracker.

Reply:
ooh, the ip tracker isn't related to k8s-local. so it was just wild goose chase...
I guess it can be removed